### ISSUE TITLE
Accessibility - Graphs missing alt text

### ISSUE THREAD

**Issue by maur1** [DependencyTrack/frontend#1200]
### Current Behavior

Graphs missing alt text or blank alt text if they can’t be described

### Proposed Behavior

Add alt text to graphs that can be described and blank if not

### Checklist

- [x] I have read and understand the [contributing guidelines](https://github.com/DependencyTrack/dependency-track/blob/master/CONTRIBUTING.md#filing-issues)
- [x] I have checked the [existing issues](https://github.com/DependencyTrack/frontend/issues) for whether this enhancement was already requested

**Comment by nscuro** [comment]
Just want to say thanks so much for reporting all these accessibility issues @maur1! We really appreciate folks with expertise looking at this.

**Comment by cuteshaun** [comment]
I'd be happy to take this on. Let me know if it's available!
